fastlane/Gemfile

```diff
@@ -22,3 +22,5 @@ end
 
 plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
 eval(File.read(plugins_path), binding) if File.exist?(plugins_path)
+
+gem 'gh-issues-inspector', git: "https://github.com/orta/gh-issues-inspector", branch: "fixed-issue-search"
```

Comment:
this is merged, if @KrauseFx makes a PR about the `require` being the same as the gem name issue then I'm happy to ship the gem to rubygems to negate this